### ISSUE TITLE
Activate microphone and camera 

### ISSUE THREAD

**Issue by Maxdux** [amanharwara/altus#305]
Hello I still have the problem to activate the microphone and the camera 
since the 4 version this client doesn't work anymore i don't understand that others can work with it 

I have tried this on different PC it does not work 
There must be an admin able to solve this problem I have been waiting for it for several months
![Screenshot_14](https://github.com/amanharwara/altus/assets/110999613/74fdd150-9ae2-4a47-9e74-3daa44d498be)
![Screenshot_15](https://github.com/amanharwara/altus/assets/110999613/621185b4-1173-4a3a-bcd4-668f0edbdf9b)

**Comment by NicolasPL64** [comment]
Yeah, I had to downgrade. v5 is a big nono for me. I'm also looking for other alternatives to Altus, lmk if you know any
